script.js

```diff
@@ -24,8 +24,14 @@ isChrome = () => {
 };
 
 headersReceivedListener = (requestDetails) => {
+  // Do not count bytes from requests from local cache
+  if (requestDetails.fromCache) return
```

Comment:
?
```suggestion
  if (requestDetails.fromCache || requestDetails.incognito) {
      return;
  }
```
And remove CONST_INCOGNITO